docs-en/chapter_data_structure/basic_data_types.md

```diff
@@ -1,24 +1,24 @@
-# Fundamental Data Types
+# Basic Data Types
 
-When we think of data in computers, we imagine various forms like text, images, videos, voice, 3D models, etc. Despite their different organizational forms, they are all composed of various fundamental data types.
+When discussing data in computers, various forms like text, images, videos, voice and 3D models comes to mind. Despite their different organizational forms, they are all composed of various basic data types.
 
-**Fundamental data types are those that the CPU can directly operate on** and are directly used in algorithms, mainly including the following.
+**Basic data types are those that the CPU can directly operate on** and are directly used in algorithms, mainly including the following.
```

Comment:
Hi Yuelinxin
these are just suggestions.
instead of '.' can be changed to ->
':'